kml/serdes_gpx.cpp

```diff
@@ -290,10 +290,17 @@ void GpxParser::Pop(std::string_view tag)
   }
   else if (tag == gpx::kTrkSeg || tag == gpx::kRte)
   {
-    CheckAndCorrectTimestamps();
+    if (m_line.size() > 1)
```

Comment:
Is the bug that OM can't render a line with one point?

Technically:
1. The attached example doesn't break GPX spec.
2. A track recording software may split recorded points into whatever segments.
3. In the context of other segments this alone point can still be counted as a valid part of the track.

We can of course just drop these points and avoid supporting some rare cases (at least for now). It would be better to make clearer comments here.

Reply:
I thought about it, but seems like better to skip (not include pt to a track), because:
- we don't connect nearby trkseg (and they are not supposed to be connected)
- even the example shows a big time/length diff between segs.